### DISCUSSION TITLE
Array to string conversion error

### DISCUSSION THREAD

**Discussion by jannysurmieda** [dbfx/laravel-strapi#5] [Q&A] [answered]
Hi... I encountered an "ErrorException Array to string conversion" Here is the screenshot of the error.... 

![Screen Shot 2021-09-04 at 4 23 54 PM](https://user-images.githubusercontent.com/25314087/132088139-8109e722-72aa-4d6d-911e-aa0ac51fe91a.png)

How can this be corrected?

**Comment by sietzekeuning** [comment]
I got exactly the same after making a Media field. Is there any news about this?

**Comment by dbfx** [comment, marked as answer]
This should be fixed now thanks to @sietzekeuning's PR
